examples/extensions/horizontal-pod-autoscaling/README.md

```diff
@@ -186,13 +175,13 @@ $ while true; do curl http://146.148.6.244; done
 In the case presented here, it increased the number of serving pods to 10:
 
 ```console
-$ kubectl get extensions/hpa
-NAME         REFERENCE                                   TARGET     CURRENT    MINPODS   MAXPODS   AGE
-php-apache   ReplicationController/default/php-apache/   100m cpu   133m cpu   1         10        15m
+$ kubectl get hpa
+NAME         REFERENCE                                   TARGET    CURRENT   MINPODS   MAXPODS   AGE
+php-apache   ReplicationController/default/php-apache/   50%       65%       1         10        14m
```

Comment:
Because of the increased number of replicas.
1 replica 305% -> 7 replicas ~45% (don't remember exact stat) -> increase load -> 10 replicas 65% (this is above the target, but we are capped by max replicas = 10)